src/build/build.ts

```diff
@@ -117,25 +94,29 @@ export async function build(
       Object.assign(bundlerOptions, options.bundle);
     }
     buildStream = buildStream.pipe(polymerProject.bundler(bundlerOptions));
-
-    if (transformModulesToAmd) {
-      buildStream = pipeStreams([
-        buildStream,
-
-        gulpif(matchesExt('.js'), new JsTransform({
-                 js: {transformModulesToAmd: true},
-                 rootDir: polymerProject.config.root,
-               })),
-
-        gulpif(matchesExt('.html'), new HtmlTransform({
-                 entrypointPath: polymerProject.config.entrypoint,
-                 js: {transformModulesToAmd: true},
-                 rootDir: polymerProject.config.root,
-               })),
-      ]);
-    }
   }
 
+  const htmlSplitter = new HtmlSplitter();
```

Comment:
Do we have a test to ensure that we're inserting the ES5 shim properly?

Reply:
The golden below shows that we did not change how we inject the es5 adapter script tag (the HTML is just now better minimized). As for whether we compile it or not, this is tested here: https://github.com/Polymer/polymer-build/blob/master/src/test/optimize-streams_test.ts#L74.